src/pull-request/pull-request-diff.html

```diff
@@ -101,7 +115,7 @@
       _removeChunk: function(e) {
         this.splice('chunks', e.model.index, 1);
       },
-      _parseDiffToChunks: (function() {
+      _parseDiffToChunks: function() {
```

Comment:
These parenthesis should not be removed. They create a scope for the regex declaration, so they are not leaking into global scope. Why did you remove them? (also for line 176)